src/vm/amd64/unixasmhelpers.S

```diff
@@ -172,6 +172,98 @@ NESTED_ENTRY JIT_RareDisableHelper, _TEXT, NoHandler
 
 NESTED_END JIT_RareDisableHelper, _TEXT
 
+#ifdef FEATURE_HIJACK
+
+//------------------------------------------------
+// OnHijackScalarTripThread
+//
+NESTED_ENTRY OnHijackScalarTripThread, _TEXT, NoHandler
```

Comment:
@kangaroo I've updated the comment to remove the part about speaking offline. Sorry about that. However, the rest of my comment stands on its own as a response. The work for passing structs in registers hasn't been merged yet (it is in a public fork), which is why we decided not to address it now for this PR.